_posts/2016-02-24-session-user-flow.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Use session variables to optimize your user flow"
+date: 2016-02-24 13:37:00
+categories: ["ruby", "rails", "sessions"]
+author: "schmierkov"
+---
+
+Sessions provide you a nice little data storage feature where the application does not need to get the informations directly from the database. So you do not have to persist data in your database and can easily store infos to a user on the fly. This is a nice way to enhance the user experience on your page. Let's say you want to show all users, from a specific area, a new fancy sign up form and the rest only sees the old form. With storing these location informations in a session variable, you do not need to persist this infos in your database.
```

Comment:
"With storing these location informations in a session variable, you do not need to persist this infos in your database." could be "If you store the version of the sign up form in a session variable, you don't need to persist this info in your database."